RogueMaster/flipperzero-firmware-wPlugins
# Restored iButton CLI

# What's new

- Restored iButton CLI functions since making iButton an external app broke CLI commands.

# Verification 

- Load up a Serial connection to FlipperZero
- Check if `ikey` and `onewire` commands are available
- Test commands

### Note:

- I could not test `onewire` command

# Checklist (For Reviewer)
